docs/source/reference/pragmas.rst

```diff
@@ -130,6 +130,18 @@ Set the expression search timeout in milliseconds.  The default is 1000.
 Set the maximum number of stuck applications allowed while unifying a meta. The
 default value is 25.
 
+``%cg``
+--------------------
+
+Codegen directives can be included in source code with the ``%cg`` pragma. For example, instead of
+using ``--directive extraRuntime=mycode.ss`` on the command line for the chez backend, you can write:
+
+.. code-block:: idris
+
+    %cg chez extraRuntime=mycode.ss
+
+See the code generator documentation for available directives.
+
```

Comment:
I think it should be added whether these directives sum up across the modules of the project, and clarify how successive pragmas behave in one module being applied to the same or different backends